anaconda_project/internal/cli/project_load.py

```diff
@@ -40,8 +40,13 @@ def load_project(dirname):
     project = Project(dirname, frontend=CliFrontend())
 
     if not os.path.exists(project.project_file.filename):
-        print("No project file exists. Please create one by running `anaconda-project init`.")
-        exit(1)
+        message = "No project file exists. Would you like to create one?"
```

Comment:
I'd like to vote against this, actually. In contexts where I am not expecting the command to be interactive---that is, it should do what it is asked with no further input from me---it should not _become_ interactive due to an error.